app/src/main/java/lvc/com/movies/database/MovieDatabase.java

```diff
@@ -0,0 +1,20 @@
+package lvc.com.movies.database;
+
+import android.arch.persistence.room.Database;
+import android.arch.persistence.room.RoomDatabase;
+import android.arch.persistence.room.TypeConverters;
+
+import lvc.com.movies.model.Movie;
+import lvc.com.movies.model.MovieSearch;
+
+/**
+ * Created by leonardo2050 on 27/01/18.
+ */
+
+@Database(entities = {Movie.class, MovieSearch.class}, version = 1)
```

Comment:
Why do you think a database is a good fit for your solution?

Reply:
I needed a local cache to avoid call a http request every time the user open the app. 
I know that Retrofit provide a cache and we can configure how long this cache will survive. 
But I believe with a local database we have more control about the data. 
So I’m able to know exactly what kind of registers do I have in my device, and I can schedule tasks to clean/update it  outside Retrofit framework.
Also, a database provides to our application all the SQL operations that could be in handy in the future.

Reply:
Great! Thanks for the explanation.